roombapy/entry_points.py

```diff
@@ -27,10 +28,18 @@ def password():
     roomba_info = roomba_discovery.find(roomba_ip)
     _validate_roomba_info(roomba_info)
 
-    roomba_password = RoombaPassword(roomba_ip)
-    found_password = roomba_password.get_password()
-    roomba_info.password = found_password
-    print(roomba_info)
+    for ciphers in ["DEFAULT@SECLEVEL=1", "DEFAULT:!DH"]:
```

Comment:
let's extract list to const.py

Reply:
actualy i think `ciphers` receive list with comma as separator, so we can do just
```
ciphers = "DEFAULT@SECLEVEL=1,DEFAULT:!DH"
```

Reply:
If that's the case and ssl actually falls back to the second one then we can just add `ciphers = "DEFAULT@SECLEVEL=1,DEFAULT:!DH"` to const and remove all other changes. :)